src/en/proposals/revolution-rework.md

```diff
@@ -0,0 +1,166 @@
+For these reasons it is necessary for the mode to conduct a complete revamp of mechanics (code), rules, lore.
+
+Also worth noting are the current flashes, which are a definite meta, because the player even by sound becomes aware of the game mode, which can affect his future decisions, which the admins obviously can't track. 
+Also flashes are an easy excuse to choke the headrevs and ruin the round.
+The forced conversion mech itself is stupid. People won't even have a choice of which side to step on, which will narrow the RP process in the mode and just turn it into a prototype of cultists, 
+getting rid of any meaning and mood of the rebellion itself.
+
+Anyways the goal of this concept is to not treat revolution as a LRP mode, thereby increasing its RP level and lowering player carelessness. 
+It's also worth considering revolution as a major mode rather than a minor mode.
+
+### Concept Abstracts
+- unique scoring system
+- voluntary entry into the revolution
+- rev icon replacement
+- additional avenues for recruitment
+- rev uplink 
+- various rebel paraphernalia
+- cult-like alert
+
+## Background
+**You can read a more comfortable and beautiful version here --->** <https://docs.google.com/document/d/1KaitkDoAZFSBrNIc78K522gtLkDHlZaMzH9vNMe9Oz4/edit?usp=sharing>
+
+It's contains additional comments, illustatrations, images and other MAYBE useful information.
+
+**RUS VERSION --->** <https://docs.google.com/document/d/11Lpip1kL9xyVAVoQn-PVuqBHv-stovz8ZaweRDGwxtE/edit?usp=sharing>
+
+## 1. GENERAL MECHANICS
+### 1.1 Rev Uplink
+- Headrevs have another distinctive feature, the rev **uplink implant**. It contains a variety of often decrepit, rusty and makeshift equipment.
+- For **one recruited** it gives **1 Rebel Telecrystal** (RTC) to headrev’s uplink.
+> A full list of gear is being worked on.
+Approximately:
+Soviet-made pistols, Mosinky, PPSh, Kalashnikovs, RGDs, various body armor, clothing, riot posters, homemade war makings, ammunition and other paraphernalia
+
+**FOR WHAT:** These things will make the headrev stand out from the crowd or help him in some minor actions for later climax (like raiding an security patrol).
+
+### 1.2 Holographic Armband
+- In exchange for the icons, the revolutionary will be given the **ability "make intentions known"** which will create a red or blue-red **armband** on his shoulder for the headrevs.
+- The **ability is one-time** and there is no way to remove this bandage once activated.
+- To know the player's affiliation before activation requires examining the character - it will answer it in examine window (being a revolutionary).
+- Once alerted _(1.6)_, all armbands automatically appear on the shoulders of the rebels.
+
+**FOR WHAT:** To get rid of stealthy revolutions, add "rebel entourage" and replace trivial meta icons.
+
+### 1.3 Mindshield
```

Comment:
I personally really hate the concept of mindshields, they are a really lazy bandaid to try and artificially slow/hard counter a conversion-based antag. It's good that your proposal nerfs them but I think that mindshields in general are unhealthy for the game and very quickly take over and become the objective of a round.